main/client.go

```diff
@@ -221,11 +221,11 @@ func main() {
 
 		doUser()
 		doReport()
-		doMap()
-		doStats()
-		doTeams()
-		doTopScores()
-		doWriteReferral()
-		doReadReferral()
-		doGenerateReferral()
+		// doMap()
```

Comment:
Why that? What's the intent of commenting out it all?

Reply:
You're looking at the previous commit. I removed comments in the last commit.